common/views/themes/utility-classes.ts

```diff
@@ -19,548 +17,544 @@ ${Object.entries(themeValues.colors)
+  }
 
-.overflow-hidden {
-  overflow: hidden;
-}
+  .round {
+    border-radius: 50%;
+  }
 
-.cursor-zoom-in {
-  cursor: zoom-in;
-}
+  .relative {
+    position: relative;
+  }
 
-// Used mainly for images
-.max-height-70vh {
-  max-height: 70vh;
-}
+  .absolute {
+    position: absolute;
+  }
 
-.width-auto {
-  width: auto;
-  max-width: 100%;
-}
+  .full-width {
+    width: 100%;
+  }
 
+  .full-height {
+    height: 100%;
+  }
 
-.clearfix {
-  ${clearfix}
-}
+  .full-max-width {
+    max-width: 100%;
+  }
 
-// TODO: use this for e.g. Promo hover behaviour too
-.card-link {
-  text-decoration: none;
+  // Images
+  .width-inherit {
+    width: inherit;
+  }
 
-  &:hover,
-  &:focus {
-    .card-link__title {
-      text-decoration: underline;
-      text-decoration-color: ${themeValues.color('green')};
+  .image-max-height-restricted {
+    max-height: 90vh;
+    max-width: 100%;
+    width: auto;
+    margin: 0 auto;
+  }
+
+  // For when we get HTML out of systems like Prismic
+  .first-para-no-margin p:first-of-type {
+    margin: 0;
+  }
+
+  // This removes the element from the flow, as well as it's visibility
+  .visually-hidden {
+    ${visuallyHidden};
+  }
+
+  .visually-hidden-focusable {
+    &:active,
+    &:focus {
+      clip: auto;
+      height: auto;
+      margin: 0;
+      overflow: visible;
+      position: static;
+      width: auto;
+      white-space: inherit;
     }
   }
-}
 
-.empty-filler:before {
-  content: "\\200b";
-}
-.shadow {
-  box-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.4);
-}
+  .opacity-0 {
+    opacity: 0;
+  }
 
-.lazy-image.lazy-image {
-  opacity: 0.6;
-  transition: opacity ${themeValues.transitionProperties};
+  .hidden {
+    visibility: hidden;
+  }
 
-  display: none;
+  .overflow-hidden {
+    overflow: hidden;
+  }
 
-  .enhanced & {
-    display: block;
+  .cursor-zoom-in {
+    cursor: zoom-in;
   }
 
-  &.lazyloaded {
-    opacity: 1;
+  // Used mainly for images
+  .max-height-70vh {
+    max-height: 70vh;
+  }
 
-    .captioned-image & {
-      opacity: 1;
+  .width-auto {
+    width: auto;
+    max-width: 100%;
+  }
+
+  .clearfix {
+    ${clearfix}
+  }
+
+  // TODO: use this for e.g. Promo hover behaviour too
+  .card-link {
+    text-decoration: none;
+
+    &:hover,
+    &:focus {
+      .card-link__title {
+        text-decoration: underline;
+        text-decoration-color: ${themeValues.color('green')};
+      }
     }
   }
 
-  .captioned-image & {
-    opacity: 0;
+  .empty-filler:before {
+    content: '\\200b';
+  }
+  .shadow {
+    box-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.4);
+  }
+
+  .lazy-image.lazy-image {
+    opacity: 0.6;
+    transition: opacity ${themeValues.transitionProperties};
+
+    display: none;
+
+    .enhanced & {
+      display: block;
+    }
+
+    &.lazyloaded {
+      opacity: 1;
+
+      .captioned-image & {
+        opacity: 1;
+      }
+    }
+
+    .captioned-image & {
+      opacity: 0;
+    }
   }
-}`;
+`;
```

Comment:
Let's not pretend I can sensibly review a file this large without my eyes glazing over; I'm going to trust this is okay.